src/H5FDs3comms.c

```diff
@@ -47,9 +47,14 @@
 
 #ifdef H5_HAVE_ROS3_VFD
 
-/* toggle debugging (enable with 1)
+/* toggle debugging: pick a level
  */
-#define S3COMMS_DEBUG 0
+#define S3COMMS_DEBUG_NONE           0
+#define S3COMMS_DEBUG_REQUESTS       1
+#define S3COMMS_DEBUG_TRACE_API      2
+#define S3COMMS_DEBUG_TRACE_INTERNAL 3
+#define S3COMMS_DEBUG_HEADERS        4
+#define S3COMMS_DEBUG                S3COMMS_DEBUG_REQUESTS
```

Comment:
Was the intention here to turn on comms debugging by default? We probably want all debugging off by default otherwise.

Reply:
no. oversight on my part. i'll push a fix asap

Reply:
thanks for catching it